apps/web/src/routes/NonAuthLayoutRouter/NonAuthLayoutRouter.js

```diff
@@ -126,24 +138,24 @@ export default function NonAuthLayoutRouter (props) {
           <Route
             path='signup/*'
             element={
-              <div className={classes.belowContainer}>
-                <Link to='/login'>
-                  {t('Already have an account?')} <Button className={classes.signupButton} color='green-white-green-border'>{t('Sign in')}</Button>
+              <div className='bg-background/100 rounded-md w-full max-w-[320px] mx-auto p-4 mt-4 text-center'>
+                <Link to='/login' className='text-foreground flex items-center justify-between gap-2'>
+                  {t('Already have an account?')} <button className='rounded-md border-2 border-foreground/20 p-2'>{t('Sign in')}</button>
```

Comment:
components/ui/button

Reply:
I've been thinking, that even if we can't get a full-blown storybook for all our UI components, it might be nice to have a `/themes` page that just visually shows the colors and a few key pieces of UI, as a dev aid. I will try crack something quick out

Reply:
yeah you can see components/ui/button-variants.tsx

i mean if the button is a one off with really its own styles/colors then do your thing, but ideally almost every button in the app would use one of the variants already defined (or new ones), so we can theme everything easily. I've been using button variants everywhere, though mostly secondary. you use a variant by specifying variant='secondary' on the button. you can also add custom styles on top of the button component too.

Reply:
We will have a storybook tom!! by the end of summer mark my words